server/src/main/java/org/elasticsearch/bootstrap/ESPolicy.java

```diff
@@ -124,11 +103,7 @@ public boolean implies(ProtectionDomain domain, Permission permission) {
             if ("<<ALL FILES>>".equals(permission.getName())) {
                 hadoopHack();
             }
-        } else if (permission instanceof RuntimePermission
-            && "getClassLoader".equals(permission.getName())
-            && isLoaderUtilGetClassLoaders()) {
```

Comment:
Just to make sure I understand. THe clue of the problem was the getParent which did not have the permission?
https://issues.apache.org/jira/browse/LOG4J2-3236

Just calling LoaderUtil.getClassLoader is safe (I suppose yes, but can you do you know why)? because it is still used 
https://github.com/apache/logging-log4j2/blob/log4j-2.18.0-rc1/log4j-api/src/main/java/org/apache/logging/log4j/util/LoaderUtil.java#L266

Reply:
Hmm... let me recheck this.

Reply:
You are right that LoaderUtil::getClassLoader is still present (in 2.18.0), and we can potentially suffer from the getParent issue which we ran into before. But it seems that all the call flows and tests do not encounter it. Since there are no plans to back port this, and all seems ok, I propose that we merge the change as is, but reserve the right to re-instate the workaround in ESPolicy if we encounter issues in the future.